packages/@react-spectrum/list/docs/ListView.mdx

```diff
@@ -361,12 +362,12 @@ This behavior is slightly different in the highlight selection style, where sing
 </Flex>
 ```
 
-{/*
+// TODO: comment this section out again
 
 ## Drag and drop
 
-To enable drag and drop in a ListView, you must provide the respective drag and drop hooks sourced from `useDnDHooks` to the ListView's `dragHooks` and `dropHooks` props respectively.
-See the examples below for various common drag and drop use cases. For more information on the hooks themselves and the various supported ways to perform a drag and drop interaction,
+To enable drag and drop in a ListView, you must provide the drag and drop hooks sourced from <TypeLink links={dndDocs.links} type={dndDocs.exports.useDnDHooks} /> to the ListView's `dndHooks` prop.
+See the examples below for various common drag and drop use cases. For more information on `useDnDHooks` and the various supported ways to perform a drag and drop interaction,
 please see the [drag and drop documentation](dnd.html).
 
```

Comment:
We may want to make some use of the `begin collapse`/`end collapse` comments to hide some of the duplicate code between these examples, and maybe also `begin highlight`/`end highlight` (added in the aria docs pr) to highlight the differences. If they aren't too large after that, we could consider showing the code by default rather than collapsing.